src/cephadm/cephadm

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/python3
 
-DEFAULT_IMAGE='docker.io/ceph/ceph:v15'
+# Use the latest stable containers also on the master branch.
+# Mainly to prevent mistakes
+DEFAULT_IMAGE='DEFAULT_IMAGE=docker.io/ceph/ceph:v15.2'
```

Comment:
use `daemon-base:latest-master-devel` to match the setting used by mgr/cephadm from the mon store?

Reply:
``DEFAULT_IMAGE='DEFAULT_IMAGE=...`` ?